Add Animal Options To Valid Animal Viewsheet (#4224)
DFHack/dfhack: docs/changelog.txt

## New Tools

## New Features

> - ``zone.animal_actions``: Add overlay to tamed animals of civ for butchering/gelding/adoption/tame(assign trainer) options in their "Overview" tab

Reviewer: ```suggestion
- `zone`: Add overlay for toggling butchering/gelding/adoption/taming options in animal "Overview" tabs
```
